src/App.js

```diff
@@ -1,19 +1,52 @@
 import React from 'react';
 import './App.css';
-
 import users from './api/users';
+import todos from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoForm } from './components/TodoForm/TodoForm';
+
+const preparedTodos = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId),
+}));
+
+class App extends React.Component {
+  state = {
+    todos: preparedTodos,
+  }
+
+  addNewTodo = (title, name) => {
+    this.setState(prevState => ({
+      todos: [
+        ...prevState.todos,
+        {
+          userId: prevState.todos.length + 1,
+          id: prevState.todos.length + 1,
+          title,
+          completed: false,
+          user: {
+            name,
+          },
+        },
```

Comment:
let first create a variable with new movie and then add it to setState call (now code looks messy)